src/SigFox.cpp

```diff
@@ -56,10 +56,11 @@ const char * sigstr[16]  =        // SIGFOX message status
 
 #define SPICONFIG   SPISettings(100000UL, MSBFIRST, SPI_MODE0)
 
-void SIGFOXClass::debug() {
+void SIGFOXClass::debug(bool ledOFF) {
```

Comment:
```suggestion
void SIGFOXClass::debug(bool const ledOFF) {
```
Please use const-correctness as much as possible.